src/ui/components/TestSuite/views/TestRecording/TestError.module.css

```diff
@@ -1,6 +1,8 @@
 .Row {
   border-left: 2px solid var(--testsuites-error-border-color);
   padding: 0.5rem 0.75rem;
+  white-space: pre-wrap;
```

Comment:
As much as possible, I'd like to try to keep changes on this branch to only affect the test suite dashboard views so merge conflicts are easier to reason about later. Mind moving the changes to `src/ui/components/TestSuite/views/TestRecording/TestError.module.css`, `src/ui/components/Tour/Tour.module.css`, and `src/ui/components/shared/Forms/TextInput.tsx` to another PR targeting `main`?